lib/index.js

```diff
@@ -111,130 +108,141 @@ function getPolyfillString(options) {
 	const ua = new UA(options.uaString);
 	const uaDebugName = ua.getFamily() + '/' + ua.getVersion() + ((ua.isUnknown() || !ua.meetsBaseline()) ? ' (unknown/unsupported; using policy `unknown='+options.unknown+'`)' : '');
 	const lf = options.minify ? '' : '\n';
-	let explainerComment = [];
+	const allWarnText = 'Using the `all` alias with polyfill.io is a very bad idea. In a future version of the service, `all` will deliver the same behaviour as `default`, so we recommend using default instead.';
+	const output = new StreamCombiner();
+	const explainerComment = [];
 
 	// Check UA and turn requested features into a list of polyfills
-	return Promise.resolve()
-		.then(() => {
-			if (options.minify) {
-				explainerComment.push('Rerun without minification (remove `.min` from URL path) for verbose metadata');
-			} else {
-				const versionStr = process.env.NODE_ENV === 'production' ? 'v'+appVersion : 'DEVELOPMENT MODE - for live use set NODE_ENV to \'production\'';
+	const supportedUA = ((!ua.meetsBaseline() || ua.isUnknown()) && options.unknown !== 'polyfill');
+	const taskPromise = supportedUA ? Promise.resolve({}) : getPolyfills(options)
```

Comment:
Speedup: previously we ran the full `always` set of feature definitions even when the user didn't specify `always`, because we need the full dependency tree for RUM.  We now do this only when rum=1 is specified.